client/src/main.tsx

```diff
@@ -32,7 +32,7 @@ const themeConfig: ThemeConfig = {
       colorPrimary: '#1cb500',
     },
     Menu: {
-      itemHeight: 60,
+      itemHeight: 30,
```

Comment:
Using about a 1.25x design pixels to real-life pixels conversion, I think we can bump this to `45` to better meet the proportions in the design